src/locale/de_DE.js

```diff
@@ -84,51 +84,51 @@ export const de_DE = {
   LBL_LEFT: 'Links',
   LBL_RIGHT: 'Rechts',
   LBL_BOTTOM: 'Unten',
-  LBL_MENU: 'Menu',
+  LBL_MENU: 'Menü',
   LBL_ERROR: 'Error',
   LBL_INFO: 'Info',
   LBL_MESSAGE: 'Message',
-  LBL_WARNINIG: 'Warning',
+  LBL_WARNINIG: 'Warnung',
   LBL_SUCCESS: 'Success',
   LBL_FAILURE: 'Failure',
-  LBL_WINDOW: 'Window',
-  LBL_WINDOWS: 'Windows',
-  LBL_NOTIFICATION: 'Notification',
-  LBL_NOTIFICATIONS: 'Notifications',
-  LBL_TRAY: 'Tray Entry',
+  LBL_WINDOW: 'Fenster',
+  LBL_WINDOWS: 'Fenster',
+  LBL_NOTIFICATION: 'Benachrichtigung',
+  LBL_NOTIFICATIONS: 'Benachrichtigungen',
+  LBL_TRAY: 'Tray-Einführung',
```

Comment:
'Tray-Eintrag'.
But how do you tanslate "Tray" into German?

Reply:
Actually, don't. Use a synonym instead:
"Infobereich-Eintrag"

A tray can be seen as a drawer of a small segment - for which you have tons of words in german. Choose a synonym instead. If I recall correctly, `LBL_TRAY` is describing the top-right section of the menubar - in which case, it could also be considered an "info(romation) section", and thus be translated as "Infobereich".

Reply:
You right "Infobereich-Eintrag" makes sense if you consider the context.

Reply:
To be honnest I didn't understand english word... :) even to update in french(perhaps because we like using English words even if we don't understand it.) Thanks !